app/keepers/keepers.go

```diff
@@ -422,6 +431,41 @@ func NewAppKeeper(
 	// Must be called on PFMRouter AFTER TransferKeeper initialized
 	appKeepers.PFMRouterKeeper.SetTransferKeeper(appKeepers.TransferKeeper)
 
+	wasmConfig, err := wasm.ReadWasmConfig(appOpts)
+	if err != nil {
+		panic("error while reading wasm config: " + err.Error())
+	}
+
+	wasmCapabilities := []string{
+		"iterator",
+		"staking",
+		"stargate",
+		"cosmwasm_1_1",
+		"cosmwasm_1_2",
+		"cosmwasm_1_3",
+		"cosmwasm_1_4",
+	}
```

Comment:
Any reason for not using `AllCapabilities()` which returns exactly that list? I mean yeah, it has a misleading doc comment and is in the wrong package. But we'll fix those minor flaws. The function itself is good IMO as it simply returns all capabilities that the current wasmd has.

Reply:
I really dislike having to use the wasmd app namespace to import this type. This was why I originally wanted it in wasmvm. tbh I think it makes sense to put in the wasmd/x/wasm/types as a const or something. Maybe I am being too picky idk

Will use `wasmapp.AllCapabilities()`

Reply:
e9058051f89db82238d6b2e17eccaa41e39adcb5